src/compute.rs

```diff
@@ -11,36 +11,76 @@ use crate::{
     util::hash_to_scalar,
 };
 
+#[allow(non_snake_case)]
+/// Compute the group commitment
+pub fn group_commitment(commitment_list: &[(Scalar, PublicNonce)]) -> Scalar {
+    let mut hasher = Sha256::new();
+    let prefix = "WSTS/group_commitment";
+
+    hasher.update(prefix.as_bytes());
+    for (id, public_nonce) in commitment_list {
+        hasher.update(id.to_bytes());
+        hasher.update(public_nonce.D.compress().as_bytes());
+        hasher.update(public_nonce.E.compress().as_bytes());
+    }
+
+    hash_to_scalar(&mut hasher)
+}
+
+#[allow(non_snake_case)]
+/// Compute the group commitment
+pub fn group_commitment_compressed(commitment_list: &[(Scalar, Compressed, Compressed)]) -> Scalar {
```

Comment:
Can we add a description of what the `commitment_list` is in these functions? Also, how are these functions different from one another? Unless they have different meaning, I would think that we refactor these functions so that one calls the other.

Or maybe get rid of `group_commitment_compressed`? I haven't checked to see if that is desirable yet.

Reply:
Ahh, looks like getting rid of the second one is undesirable.

Reply:
Yeah we need both in different contexts, unfortunately.